css-brochure-form/styles.css

```diff
@@ -0,0 +1,73 @@
+body {
+  line-height: 1em;
+  width: 90%;
+  margin: 0 auto;
+}
+
+h3 {
+  color: #ec5d57;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  text-transform: uppercase;
+  position: relative;
+  left: 150px;
+  letter-spacing: 0.1em;
+}
+
+label {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: rgb(45 45 45);
+  margin: 1em;
+  line-height: 1.5em;
+}
+
+input {
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  padding: 0.6rem;
+  width: 425px;
+  margin-top: 0.1rem;
+  margin-bottom: 1rem;
+  border-color: #cacaca;
+  border-style: solid;
+  border-width: 1.5px;
+  border-radius: 5px;
+}
+
+select {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  padding: 0.5rem;
+  width: 425px;
+  margin-top: 0.1rem;
+  margin-bottom: 0.5rem;
+  background-color: #f8f9fa;
+  border-color: #cacaca;
+  border-style: solid;
+  border-width: 1.5px;
+  border-radius: 5px;
+}
+
+.button {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 14.5px;
+  padding-top: 0.4rem;
+  padding-bottom: 0.45rem;
+  background-color: #ec5d57;
+  color: white;
+  border-radius: 40px;
+  border: 0;
+  width: 95px;
+  float: right;
```

Comment:
To overcome the issue with the floating button floating away, you can set a width on your form so that elements within don't get too wide or float away from the rest of the form's elements.

Reply:
![image](https://user-images.githubusercontent.com/103452411/186544232-7d7999cd-8f4a-4dc1-9894-d0a13aad23d8.png)
Changed it! It worked! Thank you